Content/posts/supporting-swift-58.md

```diff
@@ -0,0 +1,16 @@
+---
+date: 2023-04-03 12:00
+title: Supporting Swift 5.8
+description: Last Thursday, Apple released Swift 5.8, and today we are delighted to announce that the Swift Package Index already has full support for parsing and processing Swift packages using Xcode 14.3 and Swift 5.8 across all our supported platforms.
+---
+
+Last Thursday, [Apple released Swift 5.8](https://www.swift.org/blog/swift-5.8-released/), and today we are delighted to announce that the Swift Package Index already has full support for parsing and processing Swift packages using Xcode 14.3 and Swift 5.8 across all our supported platforms.
+
+Please feel free to [add any packages](https://swiftpackageindex.com/add-a-package) that use Swift 5.8-specific features!
+
+**Note:** For the next few days, you may see question mark icons against Swift 5.8 in the compatibility matrix. Please bear with us while our systems process the almost 30,000 builds needed to determine package compatibility with a new version of Swift!
```

Comment:
We have 66870 Swift 5.7 builds and 5.8 should be the same, i.e. I'd say "almost 67,000 builds"

Reply:
Ohh I forgot two+ branches 😳